05_iigo/index.tex

```diff
@@ -1026,31 +1026,41 @@ \subsubsection{Institutional Analysis and Development framework overview of IIGO
+
 \textbf{Diplomatic sanctions.} Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
     Suggested diplomatic sanctions include:
         \begin{itemize}
             \item Revoking an island's eligibility to vote and to be elected for a position.
             \item Revoking an island's eligibility to propose any rule or a specific rule.
         \end{itemize}
 
-\textbf{Immutable rules.} Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
-    %\item \textbf{Adding rules to the proposal list: } 
-\textbf{Scope of IIGO.} Within the scope of the coursework, additional rules could be defined to audit levels of interaction at the IITO, IIFO and foraging. \\
+\textbf{Immutable rules.} A subset of rules could be categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
```

Comment:
```suggestion
\textbf{Immutable rules.} A subset of rules could be categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been ``An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
```